pkg/provider/identity.go

```diff
@@ -368,18 +365,35 @@ func NewFullIdentity(ctx context.Context, difficulty uint16, concurrency uint) (
 	return identity, err
 }
 
+func (c ServerConfig) NewRevDB() (*peertls.RevocationDB, error) {
```

Comment:
I feel like this NewRevDB function seems out of place here but I guess it needs to be here for access to ServerConfig?

Reply:
Yea, I get where you're coming from, and yea, since it's a method on `ServerConfig` it needs to be with `ServerConfig` and I think this is the right place for that.